pkg/compute/startup.go

```diff
@@ -65,13 +68,13 @@ func (s *Startup) ensureLiveJobs(ctx context.Context) error {
 				// side-effects (particularly for ops jobs).
 				err = s.failExecution(ctx, localExecution)
 				if err != nil {
-					return err
+					errs = multierror.Append(errs, err)
 				}
 			}
 		}
 	}
 
-	return nil
+	return errs.Unwrap()
```

Comment:
`Unwrap()` only returns the first error. You want to use `ErrorOrNil()`

https://go.dev/play/p/ev8lpHV3qo3

Reply:
That'll teach me to properly read the docs and not assume it works like unwraps elsewhere :(